server.go

```diff
@@ -657,15 +657,42 @@ func wireToAddrmgrNetAddresses(netAddr []*wire.NetAddress) []*addrmgr.NetAddress
 	return addrs
 }
 
+// wireToAddrmgrNetAddressesV2 converts a collection of version 2 wire network
+// addresses to a collection of address manager network addresses.  If any
+// addresses are not able to be converted, an error is returned.
+func wireToAddrmgrNetAddressesV2(netAddr []*wire.NetAddressV2) ([]*addrmgr.NetAddress, error) {
```

Comment:
As discussed elsewhere, please use contiguous slices instead of slices to pointers.